sdks/python/apache_beam/io/vcfio.py

```diff
@@ -285,9 +285,10 @@ def __init__(self,
       try:
         self._vcf_reader = vcf.Reader(fsock=self._create_generator())
       except SyntaxError as e:
-        raise ValueError('An exception was raised when reading header from VCF '
-                         'file %s: %s' % (self._file_name,
-                                          traceback.format_exc(e)))
+        self._text_lines.throw(
```

Comment:
Please add a comment mentioning why we are invoking generator.throw() here instead of raising an error.

Reply:
Good point! Done.